src/ansys/sound/core/examples_helpers/download.py

```diff
@@ -93,17 +94,18 @@ def _retrieve_file(url, filename, _test=False):  # pragma no cover
     return local_path
 
 
-def _download_file(filename, directory=None, _test=False):  # pragma no cover
+def _get_file_path_in_dpf_server(
```

Comment:
@anshlachamb I feel that the name of the function is a bit incomplete as it also downloads the file (and all the other methods are named `download_flute, download_something` etc

What do you think of something like `_download_file_and_retrieve_path_in_dpf_server` ? A bit verbose but maybe more accurate ?

Reply:
I agree, the name is not very explicit
We should clarify the purpose of this function in its documentation

Reply:
done